app/src/main/java/com/zeapo/pwdstore/autofill/oreo/ui/AutofillPublisherChangedActivity.kt

```diff
@@ -59,13 +61,13 @@ class AutofillPublisherChangedActivity : AppCompatActivity() {
         supportActionBar?.hide()
         showPackageInfo()
 
-        okButton.setOnClickListener { finish() }
-        advancedButton.setOnClickListener {
-            advancedButton.visibility = View.INVISIBLE
-            warningAppAdvancedInfo.visibility = View.VISIBLE
-            resetButton.visibility = View.VISIBLE
+        binding.okButton.setOnClickListener { finish() }
```

Comment:
Not sure about the best practices here, but would it make sense to wrap this in a `with(binding)` block?

Reply:
It would! Fixing it right now.